dm/dm/config/task_converters.go

```diff
@@ -609,6 +612,10 @@ func SubTaskConfigsToOpenAPITask(subTaskConfigList []*SubTaskConfig) *openapi.Ta
 		task.BinlogFilterRule = &filterRuleMap
 	}
 	task.TableMigrateRule = tableMigrateRuleList
+	if len(oneSubtaskConfig.IgnoreCheckingItems) != 0 {
+		ignoreItems := oneSubtaskConfig.IgnoreCheckingItems
+		task.IgnoreCheckingItems = &ignoreItems
```

Comment:
Are there any problems if we directly assign `onSubtaskConfig.IgnoreCheckingItems` to `task.IgnoreCheckingItems`? 
```suggestion
		task.IgnoreCheckingItems = &oneSubtaskConfig.IgnoreCheckingItems
```

Reply:
because `task.IgnoreCheckingItems` is a pointer, I'm afraid it may cause `oneSubtaskConfig.IgnoreCheckingItems` is accidentally modified. here is like `ShardMode`, it's equivalent to a copy. [line 608](https://github.com/pingcap/tiflow/pull/5240/files#diff-2a04f6e529cd2dd60d67875bae5d6592b46225489691154b81cf08237e627342L605).

Reply:
Why `task.IgnoreCheckingItems` would be modified😂 A bit weird though, it's not a deal.

Reply:
i think it may be encountered in some unit test not in actual use, here is mainly for safety.